docs/en/authorization_endpoint.rst

```diff
@@ -195,57 +172,35 @@ When the **scope** parameter is used, the following values are supported:
+   The parameter **scope** MAY contain one or more values, with single spaces as separators. For example, using both profile and email in the scope parameter returns the Minimum eIDAS Dataset and the email.
+   In case of requests of single user-attributes or specific combinations of them, the RP MAY use the parameter **claims**.
+   For the definition of the parameter **claims** and its usage modes for requesting the user attributes, please refer to `OpenID.Core#ClaimsParameter`_.
 
 .. warning::
-    - If the parameter **claims** is not present or has no value and the only scope openid has been requested, only the claim *sub* is returned
-      in the response to the Userinfo Endpoint. 
-
+  If the parameter **claims** is not present or has no value and the only scope openid has been requested, only the claim *sub* is returned in the response to the *userinfo endpoint*.
+  
+.. admonition:: |spid-icon|
 
-The following table shows some usage examples.
+  The user attributes requested by **claims** parameter will be available in the *userinfo endpoint*.
+  
+.. admonition:: |cieid-icon|
 
-.. list-table:: 
-    :widths: 10 10 20 20
-    :header-rows: 1
-
-    * - **Claims**
-      - **Scope**
-      - **Attributes in the Userinfo Response**
-      - **Attributes in the ID Token**
-    * - *userinfo*: \- |br| *id_token*: \-
-      - *openid*
-      - *sub*
-      - *sub*
-    * - *userinfo*: \- |br| *id_token*: \-
-      - *openid* |br| *profile*
-      - *sub*, |br| *given_name*, |br| *family_name*, |br| *birthdate*, |br| *\https://attributes.eid.gov.it/fiscal_number*
-      - *sub*, |br| *given_name*, |br| *family_name*, |br| *birthdate*, |br| *\https://attributes.eid.gov.it/fiscal_number*
-    * - *userinfo*: \- |br| *id_token*:"birthdate":{essential:true}
-      - *openid* 
-      - *sub*, |br|  *birthdate*
-      - *sub*, |br|  *birthdate*
-    * - *userinfo*: \- |br| *id_token*: \-
-      - *openid* |br| *email*
-      - *sub*, |br| *email*, |br| *email_verified*
-      - *sub*, |br| *email*, |br| *email_verified*
-    * - *userinfo*:"family_name":null |br| *id_token*:"given_name":{essential:true}
-      - *openid* 
-      - *sub*, |br|  *family_name*, |br| *given_name* 
-      - *sub*, |br|  *given_name*
-    * - *userinfo*:\- |br| *id_token*:"birthdate":{essential:true} "gender":{essential:true}
-      - *openid* 
-      - *sub*, |br|  *birthdate*, |br|  *gender*
-      - *sub*, |br|  *birthdate*
+  The user attributes requested by **scope** or **claims** parameters will be available in both *userinfo endpoint* and *ID Token*.
+  
+.. warning::
 
+    - If the parameter **claims** is not present or has no value and the only scope openid has been requested, only the claim *sub* is returned
+      in the response to the Userinfo Endpoint.
```

Comment:
Suggerirei o di modificare in "in the response to the userinfo Endpoint and/or in the ID Token" oppure di eliminare proprio "in the response to the Userinfo Endpoint"